### ISSUE TITLE
Clean up files after running tests

### ISSUE THREAD

**Issue by nicolasdlss** [pyfsi/coconut#72]
While running the tests, files are created (specifically for the solver wrappers.
It might be cleaner to remove these files if the corresponding test has succeeded.
If the test fails, the files should remain for debugging purposes.

**Comment by toondm** [comment]
I think we can do that with the `tearDownClass`  method of the unittest module.

**Comment by hdolfen** [comment]
This also includes the restart pickle files that are written in the *`tests`* folder.

**Comment by nicolasdlss** [comment]
This has been done for the coupled solver tests and Python solver wrapper tests, using the `tearDown()` method. Note that, if `setUp()` succeeded, `tearDown()` will be run whether the test method succeeded or not.

Should still be done for the other solver wrappers. For most of them it is simply a matter of removing the working directory as follows:
```
    def tearDown(self):
        shutil.rmtree(self.working_dir)
```

**Comment by nicolasdlss** [comment]
Has been done for the Fluent tests.

**Comment by nicolasdlss** [comment]
Has been done for the OpenFOAM tests (#123 ).

**Comment by nicolasdlss** [comment]
Closed by #181
